usaspending_api/search/filters/postgres/defc.py

```diff
@@ -8,28 +8,9 @@ class DefCodes:
 
     @classmethod
     def build_def_codes_filter(cls, queryset, filter_values):
-        """Build a SQL filter to only include Subawards that are associated with any
-        DEF codes included in the API request
-
-        Args:
-            queryset (Queryset()): The existing queryset that will be added to.
-            filter_values (List[str]): List of DEF codes to use in filtering.
-
-        Returns:
-            Q(): Subquery filter containing Award IDs that are associated with the
-            provided DEF code filter(s).
-        """
```

Comment:
Please add back the docstring. I think queryset can probably be removed as a parameter as well.

Reply:
Added the docstring back and removed `queryset` as a parameter.